chainer/functions/noise/dropconnect.py

```diff
@@ -0,0 +1,116 @@
+import numpy
+
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+def _as_mat(x):
+    if x.ndim == 2:
+        return x
+    return x.reshape(len(x), -1)
+
+
+class Dropconnect(function.Function):
+
+    """Linear unit regularized by dropconnect."""
+
+    def __init__(self, dropconnect_ratio, mask=None):
```

Comment:
The argument that specifies the ratio of dropping of connection if `dropconnect_ratio` whereas that of `dropconnect` function is `ratio`. Could you consider use the same option names?